app.js

```diff
@@ -0,0 +1,12 @@
+const express = require('express')
+const app = express()
+const {getTopics} = require('./controllers/news')
+
+app.use(express.json());
+app.get('/api/topics', getTopics);
+
+app.get('*', function(req, res){
```

Comment:
Has this been tested? I do like that this was added but please only add code that has a test so we know it does work.